pages/contact.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Contact me</title>
+    <link href="../styles/style.css" rel="stylesheet" />
+    <link href="../styles/contact.css" rel="stylesheet" />
+  </head>
+  <body>
+    <nav>
+      <div class="logo">
+        <img src="/assets/logo.png" alt="Website-logo" />
+      </div>
+      <ul>
+        <li>
+          <a href="index.html"><em>00.</em>Home</a>
+        </li>
+        <li>
+          <a href="about.html"><em>01.</em>About</a>
+        </li>
+        <li>
+          <a href="portfolio.html"><em>02.</em>Projects</a>
+        </li>
+        <li>
+          <a href="contact.html"><em>03.Contact</em></a>
+        </li>
+      </ul>
+    </nav>
+    <div class="contact-container">
+      <ul class="contact-list">
+        <li>
+          <a href="https://github.com/diakymenko" target="_blank">GitHub</a>
+        </li>
+        <li>
+          <a
+            href="https://www.linkedin.com/in/daria-iakymenko-26177831"
+            target="_blank"
+            >LinkedIn</a
+          >
```

Comment:
Consider formatting multi-line tags more similarly to this for better readability: https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping